core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -21,6 +21,7 @@ import java.util.concurrent.{ConcurrentHashMap, ConcurrentLinkedQueue}
 
 import scala.collection.JavaConverters._
 import scala.collection.mutable.ArrayBuffer
+import scala.math.{max,min}
```

Comment:
It's a tiny nit, but while here, I usually see `math.max` just written out in Scala code; no need to import a standard class's methods.